ng-dev/release/publish/external-commands.ts

```diff
@@ -97,3 +97,17 @@ export async function invokeYarnInstallCommand(projectDir: string): Promise<void
     throw new FatalReleaseActionError();
   }
 }
+
+/**
+ * Invokes the `yarn check --integrity` command in order to verify up to date dependencies.
+ */
+export async function invokeYarnIntegryCheck(projectDir: string): Promise<void> {
+  try {
+    await spawn('yarn', ['check', '--integrity'], {cwd: projectDir, mode: 'silent'});
```

Comment:
Do we actually need https://classic.yarnpkg.com/en/docs/cli/check/#toc-yarn-check-verify-tree?

seems like the integrity check just checks the package json against the lock file unless I misunderstand the documentation.

Reply:
I think that makes sense, yeah 👍